modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -15,9 +15,8 @@ The {oadp-first} 1.3.0 release notes lists new features, resolved issues and bug
 
 OADP 1.3 includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses Kopia as the uploader mechanism to read the snapshot data and to write to the Unified Repository.
 
-
-//:FeatureName: Velero built-in DataMover
-Velero built-in DataMover is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+:FeatureName: Velero built-in DataMover
```

Comment:
I would move this up to be above the description.

Reply:
Sorry, I have confused myself, you mean like this
![image](https://github.com/user-attachments/assets/710da072-799f-4a63-92da-8f94d6b71292)

Thanks